dashboard/app/entities.go

```diff
@@ -123,6 +123,14 @@ type Bug struct {
 
 type BugTags struct {
 	Subsystems []BugSubsystem
+	NoRemind   BugLabelTag
+}
+
+type BugLabelTag struct {
+	// Value of the label.
+	Value bool
+	// The name of the user who last set the label.
+	SetBy string
```

Comment:
I think it's more important to remember link, or message id to create link later.
If I see a tag that I don't agree with, or that's confusing, I want to see the email that set this tag as it may contain the explanation.
If we only have the email, sending an email to that person to ask does not look handy.
The email will also contain author, date and maybe some previous discussion in the thread.